schedule/lvm_thin_provisioning.yml

```diff
@@ -0,0 +1,77 @@
+name:           lvm_thin_provisioning
+description:    >
+    Complete OS deployment with unencrypted LVM drive management.
+    Test creates 2 LVM and BIOS boot partitions.
+    Thin pool and thin lv resides on the second LVM partition,
+    where /home (XFS) is being mounted. Maintainer: mloviska, poo#39023
+conditions:
+    bootloader:
+        ARCH:
+            x86_64:
+                - installation/isosize
+                - installation/bootloader
+            s390x:
+                - installation/bootloader_zkvm
+            aarch64:
+                - installation/isosize
+                - installation/bootloader
+    addons_repos:
+        DISTRI:
+            opensuse:
+                - installation/online_repos
+                - installation/installation_mode
+                - installation/logpackages
+            sle:
+                - installation/scc_registration
+                - installation/addon_products_sle
+    hostname_inst:
+        DISTRI:
+            sle:
+                - installation/hostname_inst
+    user_settings_root:
+        DISTRI:
+            sle:
+                - installation/user_settings_root
+    reconnect_mgmt_console:
+        ARCH:
+            s390x:
+                - boot/reconnect_mgmt_console
+    sle15_workarounds:
+        DISTRI:
+            sle:
+                - console/sle15_workarounds
+    opensuse_repos:
+        DISTRI:
+            opensuse:
+                - update/zypper_clear_repos
+                - update/zypper_ar
+                - update/zypper_ref
+
+modules:
+    - {{bootloader}}
+    - installation/welcome
+    - {{addons_repos}}
+    - installation/system_role
+    - installation/partitioning
+    - installation/partitioning_lvm_thin_provisioning
+    - installation/partitioning_finish
+    - installation/installer_timezone
+    - {{hostname_inst}}
+    - installation/user_settings
+    - {{user_settings_root}}
+    - installation/installation_overview
+    - installation/disable_grub_timeout
```

Comment:
I think in this future structure we could take account to not repeat `/installation/` for every test, it might point us to something that we are doing strange.

Reply:
I also though about it, but we have majority modules from there, but then some modules from other directory, so I decided not to over-complicate it in yaml file.
One of the ideas I had additional layer, like:
```yaml
modules:
  - installation:
    - welcome
    - ...
  - boot:  
```

But that will make it much more complex for parsing, and we will have repetitive keys in case we would use hash. But in general that would be good also to visualize stages of the test suites with directories. As of now, we have it mixed, so again, if you have nice proposal for the format, please, suggest ;)

Reply:
Good idea, like using the component as a block starter, e.g.

```
installation:
 - user_settings
 - {{user_settings_root}}
 - installation_overview
 - disable_grub_timeout
```

However I would leave this for later and not included it now

Reply:
I've mentioned problem here, so nope, this won't fly =( I had exactly this idea. But for sure we can expand this approach based on our findings. As of now I would address point that we could use single yml file for tests which use some variable containing modules, like @aaannz has mentioned, not to introduce another layer of variable mappings.